data_utils/host.py

```diff
@@ -166,6 +184,9 @@ def get_hosts_from_pr_reviews(repos_file, host_dict, intern_usernames):
         for row in reader:
             if row["repo_type"] == "capstone":
                 continue
+            # Ignore repositories made in the googleinterns organization.
+            if row["owner"] == "googleinterns" and row["repo_type"] != "test":
```

Comment:
so if it's a test repo in the googleinterns repository, that's fine?

Reply:
Added clarifying comments.